assets/src/scripts/monitoring-location/components/daily-value-hydrograph/selectors/axes.js

```diff
@@ -14,6 +14,7 @@ export const getXAxis = memoize(kind =>createSelector(
         return axisBottom()
             .scale(xScale)
             .tickSizeOuter(0)
+            .ticks(7)
```

Comment:
We need to do something about the ticks but I believe there is another ticket around this to make them more reasonable. This is fine for now.

Reply:
Sounds good, I left this in for now.